examples/experimental/horizontal-pod-autoscaling/README.md

```diff
@@ -0,0 +1,189 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/experimental/horizontal-pod-autoscaling/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Horizontal Pod Autoscaler
+
+This example presents horizontal autoscaling of pods: an experimental feature delivered in version 1.1 of Kubernetes
+which automatically sets the number of pods in a replication controller (or deployment), base on CPU / memory usage.
+
+
+## Prerequisites
+
+This example requires a running Kubernetes cluster in the version 1.1 with experimental API enabled on API server (```--runtime-config=experimental/v1alpha1=true```),
+and horizontal pod autoscaler controller turned on in controller manager (```--enable-horizontal-pod-autoscaler=true```).
+This can be simply achieved on GCE by exporting ```KUBE_ENABLE_HORIZONTAL_POD_AUTOSCALER=true``` before running ```kube-up.sh``` script.
+
+The required version of kubectl is also 1.1.
+
+
+## Step One: Run & expose php-apache server
+
+To demonstrate horizontal pod autoscaler we will use a custom docker image based on php-apache server.
+The image can be found [here](image/).
+It defines [index.php](image/index.php) page which performs some CPU intensive computations.
+
+First, we will start a replication controller running the image and expose it as an external service:
+
+```console
+$ kubectl run php-apache --image=gcr.io/dev-jsz/php-apache --replicas=1
+replicationcontroller "php-apache" created
+
+$ kubectl expose rc php-apache --port=80 --type=LoadBalancer
```

Comment:
Not all clusters support LoadBalancer out of the box.  A number of folks use the Vagrant cluster to kick the tires on examples.  Maybe change to say if your cluster supports load balancing, do what you present.  

If your cluster does not support an external load balancer, create a ClusterIP service and maybe use the ```/api/v1/proxy/namespaces/default/services/php-apache``` endpoint instead to drive traffic into the cluster?